tests/02_eradiate/01_unit/radprops/test_us76_approx.py

```diff
@@ -7,37 +7,27 @@
 from eradiate.spectral.index import SpectralIndex
 
 
-@pytest.fixture
-def us76_approx_test_absorption_data_set():
-    """
-    Fixture to return the path to a test absorption data set for 'us76_approx'.
-    """
-    return eradiate.data.data_store.fetch(
-        "tests/spectra/absorption/us76_u86_4-spectra-4000_25711.nc"
-    )
-
-
-def test_us76_approx_rad_profile(mode_mono, us76_approx_test_absorption_data_set):
+def test_us76_approx_rad_profile(mode_mono):
     """
     Collision coefficient evaluation methods return pint.Quantity objects.
     """
-    p = US76ApproxRadProfile(absorption_dataset=us76_approx_test_absorption_data_set)
+    path = "spectra/absorption/us76_u86_4/us76_u86_4-spectra-18000_19000.nc"
+    p = US76ApproxRadProfile(absorption_dataset=path)
```

Comment:
Let's merge these two lines (there is no point declaring an extra variable since you don't reuse it):
```suggestion
    p = US76ApproxRadProfile(absorption_dataset="spectra/absorption/us76_u86_4/us76_u86_4-spectra-18000_19000.nc")
```